src/app/zones/locales/en-us/index.yaml

```diff
@@ -18,7 +18,11 @@ zones:
   routes:
     create:
       title: 'Create & connect Zone'
+    item:
+      title: Zone CPs
+      breadcrumbs: Zone CPs
```

Comment:
Does this even exist? Isn't it `zone-cps.routes.item` and there are already i18n entries for this? Same for items.title below.

Reply:
Oh, I didn’t notice the `zone-cps` entry existed. I’ve removed the unnecessary entries here and used the CP entry instead (for breadcrumbs).